browser/ui/tabs/split_view_tab_tile_data.h

```diff
@@ -19,8 +19,13 @@
 
 class BrowserWindowInterface;
 class SplitViewTabStripModelAdapter;
-class SplitViewBrowserDataObserver;
+class SplitViewTabTileDataObserver;
 
+// TabTile represents two tabs tied together like tile in tab strip UI.
```

Comment:
I'm not sure what tile in tab strip UI is, link?

Reply:
I also don't have any info. I assume that Tile term is picked during the implementation only in code.